instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/net/PeerServiceResolver.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.net;
+
+import java.util.Map;
+import javax.annotation.Nullable;
+
+public interface PeerServiceResolver {
+
+  public boolean isEmpty();
+
+  @Nullable
+  public String resolveService(String host, Integer port, String path);
```

Comment:
I notice that one of the invocations passes an explicit `null` for the path. Do we need that param then to also have a `@Nullable` on it?

Reply:
I don’t see the need for that annotation at all. Maybe for documentation for otherwise it doesn’t seems to enforce any behavior. 
If you think it’s necessary, feel free to add it.